python/py-fuzzer/fuzz.py

```diff
@@ -154,6 +154,25 @@ def fuzz_code(seed: Seed, args: ResolvedCliArgs) -> FuzzResult:
     """Return a `FuzzResult` instance describing the fuzzing result from this seed."""
     # TODO(carljm) remove once we debug the slowness of these seeds
     skip_check = seed in {120, 160, 335}
+    # TODO(sharkdp) the following seeds all fail due to cyclic / self-referential type aliaes or generics
+    skip_check |= seed in {
+        3,
+        10,
+        39,
+        69,
+        100,
+        110,
+        173,
+        187,
+        192,
+        215,
+        267,
+        284,
+        302,
+        311,
+        357,
+        395,
+    }
```

Comment:
I don't think this should be necessary because the fuzzer CI job currently only complains if you introduce _new_ panics relative to the `main` branch. So once this PR is merged, the new failures will be naturally incorporated into the baseline for future PRs, and the fuzzer CI job won't fail on those future PRs.

The reason for the existing `skip_check` set above isn't because we panic on those seeds -- it's because we take an excruciatingly long time attempting to resolve cycles when we analyze those seeds (or at least, that was the case last time we checked), and it sucks if the fuzzer job takes 15 minutes in CI 😄